Resources/config/security_expressions.xml

```diff
@@ -27,7 +27,7 @@
 
     <services>
         <service id="security.extra.twig_extension" class="%security.extra.twig_extension.class%" public="false">
-            <argument type="service" id="security.context" />
+            <argument type="service" id="security.authorization_checker" />
```

Comment:
I am not too experienced in symfony, how would you keep support for older versions? you just mean sticking to the 2.6 deprecated security.context, right?